care/facility/api/serializers/patient.py

```diff
@@ -33,6 +33,7 @@
     BLOOD_GROUP_CHOICES,
     DISEASE_STATUS_CHOICES,
     DiseaseStatusEnum,
+    NewDiseaseReasonEnum,
```

Comment:
typo?

Reply:
Yep 😅 Renamed to `NewDischargeReasonEnum`